src/test/scala/cookbook/RegOfVec.scala

```diff
@@ -0,0 +1,26 @@
+// See LICENSE for license details.
+
+package cookbook
+
+import chisel3._
+
+/* ### How do I create a Reg of type Vec?
+ *
+ * For information, please see the API documentation
+ * (https://chisel.eecs.berkeley.edu/api/index.html#chisel3.core.Vec)
+ */
+class RegOfVec extends CookbookTester(2) {
+  // Reg of Vec of 32-bit UInts
+  val regOfVec = Reg(Vec(4, UInt(32.W)))
```

Comment:
I was planning to put that in the api documentation which this links to, do you think it should be here as well?